python_modules/dagster/dagster/_daemon/sensor.py

```diff
@@ -1273,7 +1273,7 @@ def get_elapsed(state: InstigatorState) -> Optional[float]:
     )
 
 
-def _fetch_existing_runs(
+def fetch_existing_runs(
```

Comment:
could move this outsie of dagster._daemon and into the saem module as RemoteSensor -  slightly odd to take a new dep here in dagster-graphql (and I could possibly imagine it being the cause of import perf test failures if those are new to this PR)